executor/aggfuncs/aggfunc_test.go

```diff
@@ -45,6 +45,10 @@ import (
 	"github.com/pingcap/tidb/util/set"
 )
 
+const (
+	Separator = " "
```

Comment:
Can we start with a lowercase letter? I think this constant will only be used in this package

Reply:
Agree, we can use`const separator = " "` to instead of it. and it's better to add a comment for `separator`.